cypress/e2e/settings/theme-choice-panel.spec.ts

```diff
@@ -0,0 +1,128 @@
+/*
+Copyright 2023 Suguru Hirahara
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/// <reference types="cypress" />
+
+import { HomeserverInstance } from "../../plugins/utils/homeserver";
+import { SettingLevel } from "../../../src/settings/SettingLevel";
+
+const USER_NAME = "Hanako";
+
+describe("Theme Choice Panel", () => {
+    let homeserver: HomeserverInstance;
+
+    beforeEach(() => {
+        cy.startHomeserver("default").then((data) => {
+            homeserver = data;
+            cy.initTestUser(homeserver, USER_NAME);
+        });
+
+        // Disable the default theme for consistency in case ThemeWatcher automatically chooses it
+        cy.setSettingValue("use_system_theme", null, SettingLevel.DEVICE, false);
```

Comment:
This line is to address the inconsistency between local Cypress and Cypress on Github; the latter enables system theme by default.